### PR TITLE
Add import specifier prevent duplicates

### PR DESCRIPTION
**Pull request by cartogram**
## Description

This PR prevents duplicate specifiers from being added when using the `addImportSpecifier` transform. If duplicates end up in the source, they cause a syntax error for transforms down the line.

## Type of change

- [x]  Patch: Bug (non-breaking change which fixes an issue)
- [ ]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by cartogram** [comment]
@Shopify/web-foundations do you know what the ruby-tests failures in CI are about?

**Comment by atesgoral** [comment]
@cartogram The build issue on main is fixed.

**Comment by cartogram** [comment]
@atesgoral Thanks for following up!

Would I also be able to get a review 🍏 from you or someone on the web-foundations team?

File: packages/ast-utilities/src/javascript/addImportStatement/addImportStatement.ts [hunk 1]
```diff
@@ -19,5 +19,19 @@ export default function addImportStatement(statement: string | string[]) {
         path.node.body.unshift(ast);
       });
     },
+    File(path: traverse.NodePath<t.Program>) {
```
Comment (atesgoral) [inline packages/ast-utilities/src/javascript/addImportStatement/addImportStatement.ts:22]: Could we add a test for this case?

Comment (cartogram) [reply packages/ast-utilities/src/javascript/addImportStatement/addImportStatement.ts:22]: Good catch! I can't with my current test utilities because the `File` node is not part of the fixture (they start at `Program` because they are coming from strings not files). I tried to get around this without any luck.

I suggest we merge without a test for this and I'll come back to it. These utilities are already well tested given how experimental they are, and this particular logic exists to catch the rare edge case that they run this transform on an empty file.

What do you think?
